sdks/go/pkg/beam/io/rtrackers/wrappedbounded/wrappedbounded.go

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package offsetrange defines a restriction and restriction tracker for offset
+// ranges. An offset range is just a range, with a start and end, that can
+// begin at an offset, and is commonly used to represent byte ranges for files
+// or indices for iterable containers.
+
+package wrappedbounded
```

Comment:
TBH this seems generally useful so we can just incorporate it into the SDF package, rather than spawn a new package for a type. Packages aren't Java Classes, and this isn't a stand alone RTracker/Restriction implementation like OffsetRange.

Also, Package Doc *must* be attached without a blank line to the declaration. As written this package has no doc header.

Reply:
Moved. 

Package doc was inadvertent white space, I'm surprised the linter didn't yell at me for that one.